### PR TITLE
Resolve documentation problems identified in issues #2136, #872, #993, and #2194

### PR DESCRIPTION
**Pull request by olyson**
### Description of changes

Resolve documentation problems identified in issues #2136, #872, #993, #2194

Contributors other than yourself, if any:  @samsrabin 

CTSM Issues Fixed (include github issue #): #2136 , #872, #993, #2194

Are answers expected to change (and if so in what way)? No

Any User Interface Changes (namelist or namelist defaults changes)? No

Testing performed, if any:
Built html on local machine.

### CODE REVIEW

File: doc/source/tech_note/Crop_Irrigation/CLM50_Tech_Note_Crop_Irrigation.rst
Comment (samsrabin) [inline doc/source/tech_note/Crop_Irrigation/CLM50_Tech_Note_Crop_Irrigation.rst:540]: - [ ] Could you add back the definition of $w_{avail}$ here? It doesn't seem to be defined anywhere else.

File: doc/source/tech_note/Crop_Irrigation/CLM50_Tech_Note_Crop_Irrigation.rst
Comment (samsrabin) [inline doc/source/tech_note/Crop_Irrigation/CLM50_Tech_Note_Crop_Irrigation.rst:562]: - [ ] Looks like I missed the manual line break here. Could you please move this line up to the end of the previous one?

File: doc/source/tech_note/Crop_Irrigation/CLM50_Tech_Note_Crop_Irrigation.rst
Comment (samsrabin) [inline doc/source/tech_note/Crop_Irrigation/CLM50_Tech_Note_Crop_Irrigation.rst:569]: Great explanation!

File: doc/source/tech_note/Land-Only_Mode/CLM50_Tech_Note_Land-Only_Mode.rst
Comment (samsrabin) [inline doc/source/tech_note/Land-Only_Mode/CLM50_Tech_Note_Land-Only_Mode.rst:1]: - [ ] This change results in `undefined label: rst_land-only mode`. Could you please undo? (Or was there a reason you did this?)

File: doc/source/tech_note/Photosynthetic_Capacity/CLM50_Tech_Note_Photosynthetic_Capacity.rst
Comment (samsrabin) [inline doc/source/tech_note/Photosynthetic_Capacity/CLM50_Tech_Note_Photosynthetic_Capacity.rst:114]: - [ ] I actually refer to $N_{cb}$ in #2221. Is there a reason you got rid of it here?

**Review by samsrabin — CHANGES_REQUESTED** [review]
Thanks for all these changes, Keith! It'll be great to get these four issues resolved. I just have some minor requests before approval.

File: doc/source/tech_note/Crop_Irrigation/CLM50_Tech_Note_Crop_Irrigation.rst
Comment (olyson) [reply doc/source/tech_note/Crop_Irrigation/CLM50_Tech_Note_Crop_Irrigation.rst:540]: It is defined just below in equation 2.26.23, least in my html build.

File: doc/source/tech_note/Land-Only_Mode/CLM50_Tech_Note_Land-Only_Mode.rst
Comment (olyson) [reply doc/source/tech_note/Land-Only_Mode/CLM50_Tech_Note_Land-Only_Mode.rst:1]: Good catch.  Not sure how that happened.  Fixed now (but please double-check).

File: doc/source/tech_note/Photosynthetic_Capacity/CLM50_Tech_Note_Photosynthetic_Capacity.rst
Comment (olyson) [reply doc/source/tech_note/Photosynthetic_Capacity/CLM50_Tech_Note_Photosynthetic_Capacity.rst:114]: It's not actually a parameter in the parameter file.

File: doc/source/tech_note/Crop_Irrigation/CLM50_Tech_Note_Crop_Irrigation.rst
Comment (olyson) [reply doc/source/tech_note/Crop_Irrigation/CLM50_Tech_Note_Crop_Irrigation.rst:569]: Actually, that was @billsacks 's great explanation.

File: doc/source/tech_note/Crop_Irrigation/CLM50_Tech_Note_Crop_Irrigation.rst
Comment (olyson) [reply doc/source/tech_note/Crop_Irrigation/CLM50_Tech_Note_Crop_Irrigation.rst:562]: Looks like I caused this.  Fixed now.

**Review by samsrabin — APPROVED** [review]

**Comment by samsrabin** [comment]
Thanks, Keith!

**Pull request merged**
